feeds_migrate.module

```diff
@@ -133,3 +134,27 @@ function feeds_migrate_migrate_process_info_alter(array &$plugins) {
     }
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function feeds_migrate_form_migration_delete_form_alter(&$form, \Drupal\Core\Form\FormStateInterface $form_state, $form_id) {
+  $migration = $form_state->getBuildInfo()['callback_object']->getEntity();
+  // See which feeds migrate importers use this migration.
+  $num_feeds_migrate_importer_dependencies = 0;
```

Comment:
This seems overly long for a variable name. Maybe something short and sweet like `$dependencies` or `$dependency_count`? 🤷‍♂